Tools/autotest/rover.py

```diff
@@ -6886,6 +6887,52 @@ def REQUIRE_LOCATION_FOR_ARMING(self):
         self.context_pop()
         self.reboot_sitl()
 
+    def UART_raw_logging(self):
+        '''test raw logging of UART data'''
+        expected_log_filepath = "uart1_000.log"
+        if os.path.exists(expected_log_filepath):
+            os.unlink(expected_log_filepath)
+        self.set_parameter("SERIAL1_OPTIONS", 8192)
+        self.reboot_sitl()
+        if os.path.exists(expected_log_filepath):
+            raise NotAchievedException("Log started to exist when it shouldn't")
+        mavproxy = self.start_mavproxy()
+        self.delay_sim_time(10)
+        self.stop_mavproxy(mavproxy)
+
+        if not os.path.exists(expected_log_filepath):
+            raise NotAchievedException(f"{expected_log_filepath} does not exist")
```

Comment:
Isn't all this racey?

Reply:
Yeah, any time we use MAVProxy things get a bit racy.  I've been generous with the delays :-)

Reply:
Still, you set to enable the logging and then hope the vehicle reboots before the log gets created.